Canvas/Canvas.xcodeproj/xcshareddata/xcschemes/Canvas.xcscheme

```diff
@@ -23,7 +23,7 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
If we are in release mode, won't that hurt our ability to find out where failures were?